### PR TITLE
Log total time of request when buffering body

### PR DESCRIPTION
**Pull request by tyilo**
Currently only the time to receive the headers is logged, which is often a much smaller duration than the total duration of receiving the headers and the body.

### CODE REVIEW

**Comment by yschimke** [comment]
Seems like a reasonable change. I'll wait for any other view points before landing.

We don't actively enhance these logging classes, there is an expectation that apps copy and paste reuse this code if you need further tweaks. So don't wait on us to land and release this.

**Review by yschimke — APPROVED** [review]

**Comment by yschimke** [comment]
@swankjesse can follow up with any other feedback.

**Review by swankjesse — COMMENTED** [review]
LGTM

**Pull request merged**
